subworkflows/local/inspect_data/main.nf

```diff
@@ -0,0 +1,34 @@
+include { combineByMetaKeys } from "$projectDir/modules/local/functions"
+include { GENOME_PROPERTIES } from "$projectDir/subworkflows/local/genome_properties/main"
+include { COMPARE_LIBRARIES } from "$projectDir/subworkflows/local/compare_libraries/main"
+include { SCREEN_READS      } from "$projectDir/subworkflows/local/screen_read_contamination/main"
+
+workflow INSPECT_DATA {
+    take:
+    hifi_reads     // [ meta, hifi ]
+    hifi_histogram // [ meta, hifi_hist, ktab ]
+    hic_histogram  // [ meta, hic_hist, ktab ]
+
+    main:
+    ch_versions = Channel.empty()
+    // QC Steps
+    GENOME_PROPERTIES ( hifi_histogram )
+    COMPARE_LIBRARIES ( hifi_histogram.join( hic_histogram ) )
+    SCREEN_READS (
+        PREPARE_INPUT.out.hifi,
```

Comment:
```suggestion
        INSPECT_DATA.out.hifi,
```

Reply:
nope, that suggestion is not working. Need to check further. 

I get this error:

`Launching `/home/pippel/tests/Earth-Biogenome-Project-pilot/main.nf` [silly_magritte] DSL2 - revision: 530b58d756

    Running NBIS Earth Biogenome Project Assembly workflow.

ERROR ~ No such variable: PREPARE_INPUT

 -- Check script '/home/pippel/tests/Earth-Biogenome-Project-pilot/subworkflows/local/inspect_data/main.nf' at line: 17 or see '.nextflow.log' file for more details
`

Reply:
All channels needed within a workflow should be defined by either `take:` or created in the `main:`